unit_tests/sources/declarative/decoders/test_composite_decoder.py

```
@@ -258,6 +264,30 @@ def test_composite_raw_decoder_csv_parser_values(requests_mock, encoding: str, d
     assert parsed_records == expected_data
 
 
+@pytest.mark.parametrize("set_empty_cell_to_none", [True, False])
+def test_composite_raw_decoder_parse_empty_strings(requests_mock, set_empty_cell_to_none: bool):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)